tools/accuracy_checker/README.md

```diff
@@ -97,6 +97,7 @@ You may refer to `-h, --help` to full list of command line options. Some optiona
 - `--num_requests` number requests for async execution. Allows override provided in config info. Default is `AUTO`
 - `--model_attributes` directory with additional models attributes.
 - `--subsample_size` dataset subsample size.
+- `--shuffle` allow shuffle annotation during creation a subset.
```

Comment:
please say  2-3 wordsabout default value and that it works only if subsample_size enabled

Reply:
Done